### PR TITLE
1566 shape feature extract

### PR DESCRIPTION
**Pull request by raylim**

### CODE REVIEW

**Comment by armaank** [comment]
This looks very good, good call on computing log ratios

A few items to address:
1. You can remove all features extracted for `Detection probability`, these are unneeded and can be dropped from the fx extraction phase
2. You can remove features associated w/ unclassified regions, those can be dropped from the fx extraction phase
3. The whole slide features are redundant in this output format, since they are the same no matter the cell type, so we have double the number of features for whole slide features
4. A lot of the whole slide features we probably don't need, here is a list of features that don't need to be extracted: anything related to bounding boxes (bbox), anything related to centroids, anything related to inertia tensors, whole slide label, anything related to moments, whole slide area in microns (we already have whole slide area measured in pixels, and it kind of makes sense to keep all of our measurements in pixels rather than just converting area to units of microns)


I also think it makes sense to restructure the output format. Even though it'll be a little wide, it's more convenient for downstream use to have all of these features in a single column, so the columns will be structured as `variable_parent_class` . That way for an entire cohort we can concat all of the single-row results into a dataframe suitable for downstream use in a scikit-learn or pytorch model.

**Review by tomp — APPROVED** [review]
Approved
